icons/bookmark-add.svg

```diff
@@ -0,0 +1,15 @@
+<svg
+  xmlns="http://www.w3.org/2000/svg"
+  width="24"
+  height="24"
+  viewBox="0 0 24 24"
+  fill="none"
+  stroke="currentColor"
+  stroke-width="2"
+  stroke-linecap="round"
+  stroke-linejoin="round"
+>
+  <path d="M19 21l-7-5-7 5V5a2 2 0 012-2h10a2 2 0 012 2z" />
+  <line x1="12" x2="12" y1="7" y2="12" />
+  <line x1="14.5" x2="9.5" y1="9.5" y2="9.5" />
```

Comment:
```suggestion
  <path d="m19 21-7-4-7 4V5a2 2 0 0 1 2-2h10a2 2 0 0 1 2 2v16Z"/>
  <line x1="12" x2="12" y1="7" y2="13" />
  <line x1="15" x2="9" y1="10" y2="10" />
```

Reply:
Well actually I copied from the bookmark icon and just added the plus icon. Do I need to change the `bookmark` too?

Reply:
Yes if you can do 😃, that would be great.
And maybe you can add a `bookmark-minus` as wel.

Reply:
Sure.

Reply:
✅ Done.